src/parser/jobs/gnb/modules/GnashingFang.tsx

```diff
@@ -36,19 +33,16 @@ class GnashingComboState {
 
 }
 
-export class AmmoCombo extends Analyser {
+export class GnashingFang extends Analyser {
 	static override handle = 'Gnashing Fang Combo issues'
 	static override debug = false
 
-	@dependency private checklist!: Checklist
 	@dependency private timeline!: Timeline
 	@dependency private data!: Data
 
-	private buffs = 0
-	private actions = 0
 	private errors = 0
 
-	private COMBO_BREAKERS = [this.data.actions.KEEN_EDGE.id, this.data.actions.BRUTAL_SHELL.id, this.data.actions.SOLID_BARREL.id] // These skills will break the Gnashing combo
+	private COMBO_BREAKERS = [this.data.actions.KEEN_EDGE.id, this.data.actions.BRUTAL_SHELL.id, this.data.actions.SOLID_BARREL.id, this.data.actions.LION_HEART.id, this.data.actions.NOBLE_BLOOD.id, this.data.actions.REIGN_OF_BEASTS.id] // These skills will break the Gnashing combo
```

Comment:
Was this confirmed by anyone on the media tour? It's probably not wrong, but if it's not confirmed then it might be worth chucking a TODO in as a reminder to test it and update the code if needed.

Reply:
Media tour confirmed, but will test